gcpauth/test/secretManager_test.go

```diff
@@ -0,0 +1,22 @@
+package test
+
+import (
+	"os"
+	"testing"
+
+	"github.com/Flahmingo-Investments/helpers-go/gcpauth"
+)
+
+// This test assumes that either gcloud is setup or you are working inside of
+func TestGetSecretByName(t *testing.T) {
+	secretName := os.Getenv("SecretPath")
+	secret, err := gcpauth.GetSecretByName(secretName)
+
+	if err != nil {
+		t.Error(err)
+	}
+
+	if secret != "test-value" {
+		t.Error("wrong value")
+	}
+}
```

Comment:
Please move this file together `secretManager.go`

If you don't run the test. Just add build tag.

```go
// +build integration

package somename
```

Then we can run integration test
```bash
go test -tags integration path/to/test
```